packages/icons/metadata.yml

```diff
@@ -2,13 +2,10 @@ icons:
   - name: add
     friendly_name: Add
     usage: This is a description for usage
-    categories:
-      - Navigation
+    category: Actions
```

Comment:
Are we switching to one category? Let me know if the plan changed, but I thought we were keeping a list of categories

Reply:
@joshblack after talking with @chrisconnors-ibm , it sounded like the desire was that each icon would only be categorized in 1 place, bc it's supposed to map to the sketch menu and not be duplicated. so for this PR i switched to storing the properties as "category" and "subcategory"

Reply:
Right, I think in Sketch we're limited to their "primary category" but I think we still want a list for the site in case we want to support multiple categories (as I understand it).

In other words:

```
- name: add
  categories: 
    - Primary category (used in Sketch)
    - Secondary category
```

Reply:
oh, we can't have like a sub-menu in sketch? dang sorry that surprised me!

Reply:
we can but we are trying to minimize the number of sub menus people have to navigate to get to the actual symbol. Hierarchical menus are painful - we don't want to add additional intermediate ones. 
![image](https://user-images.githubusercontent.com/35537391/59692656-0de41f80-91ab-11e9-946e-f7273b8d2eed.png)

Reply:
i see! thank you